### PR TITLE
Replace homepage screenshots

### PR DESCRIPTION
**Pull request by AshtakaOOf**
Update screenshots to 6.1, straightforward enough.
~~Also remove Customize Dark and Light and replace with only one screenshot with Fluent.~~
I hope I didn't break anything, because if I did I'm gonna cry.

### CODE REVIEW

**Comment by AshtakaOOf** [comment]
Yeah I broke something sadly

**Comment by flowln** [comment]
The update mods screenshots are not from the update mods thingy, they're from the mod downloader confirmation dialog :p

**Comment by AshtakaOOf** [comment]
AAAAAAh

**Comment by AshtakaOOf** [comment]
Flow flow I fixed it!

**Review by flowln — APPROVED** [review]
good job
 _pats Ashtaka_
 
 i'll not merge this rn since i remember that we have had issues with the image dimensions on Flatpak because of something to do with those images. I'll wait until someone with more knowledge about it takes a look at this

**Comment by DioEgizio** [comment]
Can you add a modpack updating screenshot?
 this hyperlink broke btw

**Comment by DioEgizio** [comment]
> good job _pats Ashtaka_
> 
> i'll not merge this rn since i remember that we have had issues with the image dimensions on Flatpak because of something to do with those images. I'll wait until someone with more knowledge about it takes a look at this

as #196 says, you have to edit that file in the prism repo too

File: src/index.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ hero:
       <div>
         <h1>Lightweight & Customizable</h1>
         <div class="subtitle">
-          You don't need an embedded web browser for a game launcher. Prism Launcher uses the Qt toolkit, which requires little system resources. Prism Launcher also supports customizing the instance window and theme.
+          You do not need an embedded web browser as a game launcher. Prism Launcher uses the Qt toolkit, which requires little system resources. Prism Launcher also let you customize the menu and supports [custom themes](https://prismlauncher.org/wiki/getting-started/change-themes/).
```
Comment (Scrumplex) [inline src/index.md:69]:
```suggestion
          You don't need an embedded web browser as a game launcher. Prism Launcher uses the Qt toolkit, which requires little system resources. Prism Launcher also lets you customize the menu and supports [custom themes](https://prismlauncher.org/wiki/getting-started/change-themes/).
```

**Comment by TheKodeToad** [comment]
Do the dark theme screenshots need to be duplicated?

**Comment by DioEgizio** [comment]
Why?

**Comment by AshtakaOOf** [comment]
I'm doing the modpack updating thing @DioEgizio

**Review by DioEgizio — APPROVED** [review]

**Comment by TheKodeToad** [comment]
> Why?

I mean they are duplicated.

**Comment by AshtakaOOf** [comment]
There are no cool light mode theme :(

**Comment by TheKodeToad** [comment]
> There are no cool light mode theme :( And I couldn't figure out how to have only one file...

Ah, gotcha. Maybe it would be more convenient to find a way to not show the version.

**Review by Scrumplex — COMMENTED** [review]
Why was `src/img/Rory_11.png` added? :O

**Comment by AshtakaOOf** [comment]
Because the multimc cat is also in the files :)

**Comment by DioEgizio** [comment]
> Because the multimc cat is also in the files :)

Look at the 404 page to see why

**Comment by AshtakaOOf** [comment]
I know should I replace it with Rory?

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**